src/other/broadcasting.jl

```diff
@@ -14,6 +14,8 @@ Base.Broadcast.BroadcastStyle(::Type{<:AbstractDataFrame}) =
 
 Base.Broadcast.BroadcastStyle(::DataFrameStyle, ::Base.Broadcast.BroadcastStyle) =
     DataFrameStyle()
+Base.Broadcast.BroadcastStyle(::DataFrameStyle, ::Base.Broadcast.Unknown) =
```

Comment:
I think this is normally never called AFAICT but we want to avoid introducing dispatch ambiguities in broadcasting.